data/duke.txt

```diff
@@ -0,0 +1,2 @@
+todo play true
```

Comment:
data.txt should be empty since user would not have added any task initially when they start using Duke